docs/git/pull-request-template.md

```diff
@@ -0,0 +1,63 @@
+---
+id: pull-request-template
+title: Pull Request Template
+---
+
+
+# Generic Pull Request Template
```

Comment:
If there are schema changes, section for details regarding the new schema changes would also be great.